server/sonar-server/src/main/java/org/sonar/server/permission/PermissionTemplateService.java

```diff
@@ -144,8 +157,12 @@ protected void doExecute(Long templateId, String permission) {
     updater.executeUpdate();
   }
 
+  /**
+   * @deprecated since 5.2 can be removed when Ruby doesn't rely on PermissionTemplateService
+   */
+  @Deprecated
   public void removeGroupPermission(String templateKey, String permission, String groupName) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)